source/drivers/driver-compatibility-reference.txt

```diff
@@ -1077,27 +1183,69 @@ MongoDB Compatibility
      - MongoDB 2.4
      - MongoDB 2.6
      - MongoDB 3.0
+     - MongoDB 3.2
+     - MongoDb 3.4
```

Comment:
@ajdavis can you comment on this? Should we clarify that Motor supports MongoDB 3.4 when using PyMongo 3.4? And is using PyMongo 3.4 enough to fully support all new features in MongoDB 3.4?